packages/core/src/shared/telemetry/vscodeTelemetry.json

```diff
@@ -1,5 +1,44 @@
 {
     "types": [
+        {
```

Comment:
https://github.com/aws/aws-toolkit-common/pull/1028

Reply:
that was merged so can remove this here